translate/src/utils/message/notranslate.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Adds `<span translate="no">` wrappers around `{braced}` content in `str`
```

Comment:
Could we instead do that on the backend instead?

What is the plan for other files formats? Should we use react-content-marker?

Reply:
Sure, that's possible. At the moment, the API for these endpoints has the message parsing happen in the frontend, and only the translatable pattern is included in the GET request. If we want the parsing and transform to happen in the backend, the machine translation APIs need to evolve further. I have not included that work in the current PR, and only added the `format=html|text` query parameter.

My preference would be for us to have an idea for what's happening with the machinery UI before changing its API further, separately from this PR.